rtil/src/threads/ue/rebo/livesplit.rs

```diff
@@ -43,3 +53,46 @@ impl Timer {
         LIVESPLIT_STATE.lock().unwrap().timer.set_game_time(TimeSpan::from_seconds(time));
     }
 }
+impl Run {
+    pub fn game_name() -> String {
+        LIVESPLIT_STATE.lock().unwrap().timer.run().clone().game_name().to_string()
+    }
+    pub fn set_game_info(game: Game, category: String, new_game_glitch: bool) {
+        let mut state = LIVESPLIT_STATE.lock().unwrap();
+        let mut run = state.timer.run().clone();
```

Comment:
Do you actually need to modify the existing run, or would it be fine to just create a fresh new `Run`?